src/deploy/NVA_build/builder.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:8 
+FROM registry.centos.org/centos:8
```

Comment:
Not a CI image, We need to keep it `centos:8`

Reply:
Well, this way it is clear where the image comes from, it is a recommended practise to include the registry. A default search for pulling the image may contain something else than docker.io and provide some (old/fake/..) centos image. So, IMHO it is better to pull the image from the official registry instead.